### PR TITLE
migrating .net5

### PR DESCRIPTION
**Pull request by hhblaze**

### CODE REVIEW

File: NetJSON/NetJSON.cs [hunk 1]
```diff
@@ -309,7 +309,7 @@ private static class NetJSONCachedSerializer<T> {
 
             private static NetJSONSerializer<T> GetSerializer()
             {     
-#if NET_STANDARD_20
+#if NET_STANDARD_20 || NET5_0
```
Comment (rpgmaker) [inline NetJSON/NetJSON.cs:312]: This is the part I missed.

**Review by rpgmaker — APPROVED** [review]

**Comment by hhblaze** [comment]
image

[Comparing here](https://github.com/rpgmaker/NetJSON/pull/236/commits/2bcc3b5c95ccb375926f791512a2b1502384372c)

**Comment by rpgmaker** [comment]
The other changes different is using the predefined constant vs defining mine since I did not know NET5_0 was already defined for net 5.0. But I suppose you are right, I was just counting the main problem which was using the dynamic provider instead of the assembly generator based solution which was causing the access denied exception.

**Comment by hhblaze** [comment]
Thank you, it works now!

**Pull request merged**